test/sequential/test-https-set-timeout-server.js

```diff
@@ -56,83 +57,81 @@ function run() {
 }
 
 test(function serverTimeout(cb) {
-  const server = https.createServer(serverOptions, function(req, res) {
+  const server = https.createServer(serverOptions, common.mustCall(function(req, res) {
```

Comment:
Nit: I think the linter is not happy with this as this line it too long. You can run `make jslint` to run the linter.